resources/eventing/charts/controller/values.yaml

```diff
@@ -15,7 +15,7 @@ image:
   # name is the name of the container image for the eventing-controller
   name: "eventing-controller"
   # tag is the container tag of the eventing-controller image
-  tag: "81e3637e"
+  tag: "PR-10674"
```

Comment:
Can you please use the new liveness and readiness endpoints in the controller deployments. E.g. [nats-controller](https://github.com/grischperl/kyma/blob/define-readiness-liveness-for-eventing-controller-9769/resources/eventing/charts/nats-controller/templates/deployment.yaml#L2)

Reply:
Turns out [this change](https://github.com/kyma-project/kyma/pull/11050/files#diff-236cf4a612aab7ea5de05440a447d5f33dee7a4d17a008749378273c7b482c67R39) will go in first hence we gotta clean it up with the new endpoints.